src/pybrake/failed_stats.py

```diff
@@ -0,0 +1,30 @@
+import time
+from collections import deque
+from threading import Timer
+
+from . import metrics
+
+
+class FailedStats(Timer):
```

Comment:
It would be nice to add tests.

Reply:
We should probably call `cancel()` on a timer.

Alternatively, you can inherit from `Thread` since you use a while loop https://stackoverflow.com/a/9812806/511205

I'm not a Python expert, so it's up to you.

Reply:
I'm working on test cases. Once all adjustments were approved, I thought an additional PR would be raised for that.